app/templates/courses/teacher-courses-view.jade

```diff
@@ -71,7 +71,8 @@ block content
                             = " "
                             span
                               = i18n(level.attributes, 'name').replace('Course: ', '')
-              a.play-level-button.btn.btn-lg.btn-primary
+              - var disablePlayButtons = !view.paidTeacher && features.china && course.get('campaignID') != "55b29efd1cd6abe8ce07db0d" && course.get('campaignID') != "5789236960deed1f00ec2ab8" && course.get('campaignID')  != "578913f2c8871ac2326fa3e4"
+              a.play-level-button.btn.btn-lg.btn-primary(disabled=disablePlayButtons)
```

Comment:
Why are we making it possible to disable this button?
Do we want to block teachers from accessing the other campaigns if they haven't paid?